### PR TITLE
Update to commit page JS and status icons

### PR DESCRIPTION
**Pull request by richmahn**
Just some changes for the commit pages so they display a bit better.

---
This change is [Reviewable](https://reviewable.io/reviews/unfoldingword-dev/door43.org/187)

### CODE REVIEW

File: _layouts/bible.html [hunk 1]
```diff
@@ -23,7 +23,7 @@ <h1 itemprop="headline">
                 <div id="sub-header">
                     <span id="last-updated">Updated ? hours ago</span>
                     <span id="views"><i class="fa fa-eye"></i></span>
-                    <span id="num-of-views">? views</span>
+                    <span id="num-of-views">2 views</span>
```
Comment (phillip-hopper) [inline _layouts/bible.html]: Why is this being set to 2? I think it should be blank by default.

File: _layouts/obs.html [hunk 2]
```diff
@@ -23,7 +23,7 @@ <h1 itemprop="headline">
                 <div id="sub-header">
                     <span id="last-updated">Updated ? hours ago</span>
                     <span id="views"><i class="fa fa-eye"></i></span>
-                    <span id="num-of-views">? views</span>
+                    <span id="num-of-views">2 views</span>
```
Comment (phillip-hopper) [inline _layouts/obs.html]: Same question.

File: js/commit-page.js [hunk 3]
```diff
@@ -3,10 +3,30 @@
...
+        $(this).removeClass('starred');
+        $(this).find('i').removeClass('fa-star').addClass('fa-star-o');
+    }
+    else {
+        $(this).addClass('starred');
+        $(this).find('i').removeClass('fa-star-o').addClass('fa-star');
+    }
+  });
+
+  var filename = window.location.href.split('?')[0].split('/').pop()
```
Comment (phillip-hopper) [inline js/commit-page.js]: Is a semicolon missing here?

File: _layouts/bible.html (hunk 1 above)
Comment (richmahn) [inline _layouts/bible.html]: Just to look like something for demoing proposes

Comment (richmahn) [inline _layouts/bible.html]: Just to look like something for demoing proposes

File: js/commit-page.js (hunk 3 above)
Comment (richmahn) [inline js/commit-page.js]: Oops. Been programming in python too much.

Comment (richmahn) [inline js/commit-page.js]: Oops. Been programming in python too much.

**Pull request merged**